02/app.js

```diff
@@ -36,17 +36,24 @@ function validation(e) {
 	if (!validateEmail(email)) {
 		formEmail.style.color = 'red';
 		isValid = false;
+	} else {
+		formEmail.style.color = 'black';
 	}
 
 	if (!validatePasswords(pass1, pass2)) {
 		formPass1.style.color = 'red';
 		formPass2.style.color = 'red';
 		isValid = false;
+	} else {
+		formPass1.style.color = 'black';
+		formPass2.style.color = 'black';
 	}
 
 	if (!checkbox) {
 		formCheckbox.style.color = 'red';
 		isValid = false;
+	} else {
+		formCheckbox.style.color = 'black';
 	}
 
 	if (isValid) {
```

Comment:
Można też do tego rozwiązania podejść inaczej (ale nie trzeba zmieniać tego co jest).
Sprawdzać poprawność danych. Jeśli są nieprawidłowe to do tablicy zapisywać, element, który jest nieprawidłowy.
Następnie wszystkie elementy resetować na "black".
Potem zrobić pętlę po tablicy i zaznaczać te elementy jako "red".

Przy takim rozwiązaniu będzie mniej kodu :)